<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutlineModify_Dialog</class>
 <widget class="QDialog" name="OutlineModify_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>212</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Outline Add</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
	background-color: rgb(43, 43, 43);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>208</width>
       <height>185</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>185</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Group ID :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="GroupID_lineEdit">
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1240</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 10pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="OutlineID_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Loop ID :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="LoopID_lineEdit">
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1240</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 10pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="MatID_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Mat. ID :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="MatID_lineEdit">
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1240</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 10pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="Type_label">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Solid_radioButton">
          <property name="text">
           <string>Solid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Opening_radioButton">
          <property name="text">
           <string>Opening</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="horizontalSpacing">
         <number>32</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="PointIID_label">
          <property name="text">
           <string>Points  :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="Points_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1240</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 10pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>(e.g. 1,2,3,4)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>2080</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OutlineAdd_button">
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OutlineAddCancel_pushButton">
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
